cloudflare/resource_cloudflare_teams_list_test.go

```diff
@@ -0,0 +1,65 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccCloudflareTeamsListBasic(t *testing.T) {
```

Comment:
this test is currently failing

```
TF_ACC=1 go test $(go list ./...) -v -run "^TestAccCloudflareTeams" -timeout 120m -parallel 1
?   	github.com/cloudflare/terraform-provider-cloudflare	[no test files]
=== RUN   TestAccCloudflareTeamsListBasic
    testing.go:705: Step 0 error: Check failed: Check 5/5 error: cloudflare_teams_list.ffyuqmxemk: Attribute 'items.0' expected "asdf-1234", got "asdf-5678"
--- FAIL: TestAccCloudflareTeamsListBasic (3.35s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/cloudflare	3.910s
?   	github.com/cloudflare/terraform-provider-cloudflare/version	[no test files]
FAIL
make: *** [testacc] Error 1
```

suspect this may be a bug with constructing the list items.

Reply:
Interestingly, the API returns the list items in reverse order.